ec2/spark_ec2.py

```diff
@@ -71,6 +71,7 @@
     "1.3.0",
     "1.3.1",
     "1.4.0",
+    "1.5.0"
```

Comment:
And we need to update the spark tachyon map -- i think we use Tachyon 0.7.1 in Spark 1.5.0